### PR TITLE
Type-5 Workflow Support

### PR DESCRIPTION
**Pull request by lispandfound**

### CODE REVIEW

File: workflow/automation/org/nesi/srf_gen.sl [hunk 1]
```diff
@@ -11,12 +11,12 @@ if [[ -n ${CUR_ENV} && ${CUR_HPC} != "mahuika" ]]; then
     source $CUR_ENV/workflow/workflow/environments/helper_functions/activate_env.sh $CUR_ENV "mahuika"
 fi
 
-REL_CSV=${1:?REL_CSV argument missing}
+REL_FILEPATH=${1:?REL_CSV argument missing}
```
Comment (sungeunbae) [inline workflow/automation/org/nesi/srf_gen.sl:14]: Should this be REL_YAML? It is coincidentally consistent with https://github.com/ucgmsim/slurm_gm_workflow/blob/master/workflow/automation/org/kisti/srf_gen.pbs 
SLURM variables are positional whereas PBS cares about actual variable names. Also edit line 4 above
```
# must be run with sbatch srf_gen.sl [REL_CSV] [MGMT_DB_LOC] [REL_NAME]
```

**Review by sungeunbae — CHANGES_REQUESTED** [review]

File: workflow/automation/org/kisti/srf_gen.pbs [hunk 2]
```diff
@@ -33,15 +33,21 @@ echo $start_time
...
 
 end_time=`date +$runtime_fmt`
 echo $end_time
 
-INFO_PATH=${REL_YAML%.*}.info
-STOCH_PATH=${REL_YAML%.*}.stoch
-SIM_PARAMS_PATH=${REL_YAML%.*}.yaml
+INFO_PATH=${REL_FILEPATH%.*}.info
+STOCH_PATH=${REL_FILEPATH%.*}.stoch
+SIM_PARAMS_PATH=${REL_FILEPATH%.*}.yaml
```
Comment (sungeunbae) [inline workflow/automation/org/kisti/srf_gen.pbs:50]: If REL_FILEPATH is already yaml, what happens here?

File: workflow/automation/org/nesi/srf_gen.sl [hunk 3]
```diff
@@ -33,15 +33,24 @@ echo $start_time
...
 
 end_time=`date +$runtime_fmt`
 echo $end_time
 
-INFO_PATH=${REL_CSV%.*}.info
-STOCH_PATH=${REL_CSV%.*}.stoch
-SIM_PARAMS_PATH=${REL_CSV%.*}.yaml
+INFO_PATH=${REL_FILEPATH%.*}.info
+STOCH_PATH=${REL_FILEPATH%.*}.stoch
+SIM_PARAMS_PATH=${REL_FILEPATH%.*}.yaml
```
Comment (sungeunbae) [inline workflow/automation/org/nesi/srf_gen.sl:53]: As above, if REL_FILEPATH is already yaml...?

**Review by claudio525 — APPROVED** [review]

**Review by joelridden — APPROVED** [review]

File: workflow/automation/org/kisti/srf_gen.pbs (hunk 2 above)
Comment (lispandfound) [reply workflow/automation/org/kisti/srf_gen.pbs:50]: Problems! We'll have to talk about what to do with these files in a way that doesn't completely ruin everything. If we change the sim params path here we'll need to change everything downstream...

Comment (lispandfound) [reply workflow/automation/org/kisti/srf_gen.pbs:50]: Actually, reading this file I am not even sure what `SIM_PARAMS` even is?

**Review by AndrewRidden-Harper — DISMISSED** [review]

**Review by AndrewRidden-Harper — APPROVED** [review]

**Review by joelridden — APPROVED** [review]

**Review by sungeunbae — APPROVED** [review]

**Review by claudio525 — APPROVED** [review]

**Pull request merged**